docs/extend/modular-resources/examples/rplidar.md

```diff
@@ -1,17 +1,16 @@
 ---
-title: "Add an RPlidar as a Modular Component"
-linkTitle: "Add an RPlidar as a Modular Component"
+title: "Add an RPlidar as a Modular Resource"
+linkTitle: "RPlidar"
 weight: 40
 type: "docs"
-description: "How to add an RPlidar as a modular component of your robot."
+description: "How to add an RPlidar as a modular resource of your robot."
 image: "/program/img/modular-resources/rplidar-on-robot.png"
 imageAlt: "An R-P-lidar mounted to a Viam rover."
 images: ["/program/img/modular-resources/rplidar-on-robot.png"]
-tags: ["slam", "services"]
+tags: ["slam", "services", "modular resources", "lidar", "rplidar"]
```

Comment:
remember to add an alias when you rename or move files!

Reply:
Will do, added